examples/ChatQnA/deploy/k8s_helm.md

```diff
@@ -0,0 +1,573 @@
+
+### Clone Repository 
+Next step is to clone the GenAIInfra which is the containerization and cloud native suite for OPEA, including artifacts to deploy ChatQnA in a cloud native way.
+
+```bash
+git clone https://github.com/opea-project/GenAIInfra.git
+```
+Checkout the release tag
+```
+cd GenAIInfra/helm-charts/
+git checkout tags/v1.0
+```
+### HF Token
+The example can utilize model weights from HuggingFace and langchain.
+
+Setup your [HuggingFace](https://huggingface.co/) account and generate
+[user access token](https://huggingface.co/docs/transformers.js/en/guides/private#step-1-generating-a-user-access-token).
+
+Setup the HuggingFace token
+```
+export HF_TOKEN="Your_Huggingface_API_Token"
+```
+
+### Proxy Settings
+
+For services requiring internet access, such as the LLM microservice, embedding service, reranking service, and other backend services, proxy settings can be essential. These settings ensure services can download necessary content from the internet, especially when behind a corporate firewall.
+Proxy can be set in the `values.yaml` file, like so:
+Open the `values.yaml` file using an editor
+```bash
+vi chatqna/values.yaml
+```
+Update the following section and save file:
+```yaml
+global:
+  http_proxy: "http://your-proxy-address:port"
+  https_proxy: "http://your-proxy-address:port"
+  no_proxy: "localhost,127.0.0.1,localaddress,.localdomain.com"
+```
+## Use Case Setup
+The `GenAIInfra` repository utilizes a structured Helm chart approach, comprising a primary `Charts.yaml` and individual sub-charts for components like the LLM Service, Embedding Service, and Reranking Service. Each sub-chart includes its own `values.yaml` file, enabling specific configurations such as Docker image sources and deployment parameters. This modular design facilitates flexible, scalable deployment and easy management of the GenAI application suite within Kubernetes environments. For detailed configurations and common components, visit the [GenAIInfra common components directory](https://github.com/opea-project/GenAIInfra/tree/main/helm-charts/common).
+
+This use case employs a tailored combination of Helm charts and `values.yaml` configurations to deploy the following components and tools:
+|use case components | Tools |   Model     | Service Type |
+|----------------     |--------------|-----------------------------|-------|
+|Data Prep            |  LangChain   | NA                       |OPEA Microservice |
+|VectorDB             |  Redis       | NA                       |Open source service|
+|Embedding            |   TEI        | BAAI/bge-base-en-v1.5    |OPEA Microservice |
+|Reranking            |   TEI        | BAAI/bge-reranker-base   | OPEA Microservice |
+|LLM                  |   TGI        |Intel/neural-chat-7b-v3-3 |OPEA Microservice |
+|UI                   |              | NA                       | Gateway Service |
+Tools and models mentioned in the table are configurable either through the
```

Comment:
Add a blank line after a table. Otherwise, this text will be included in the table, as shown if you viewed this file:

![image](https://github.com/user-attachments/assets/fd807dbc-cdbf-4603-9213-81676256e925)